src/main/java/io/smartcat/migration/MigrationEngine.java

```diff
@@ -43,8 +47,8 @@ public Migrator(final Session session) {
         }
 
         /**
-         * Method that executes all migration from migration resources that are higher version than db version. If
-         * migration fails, method will exit.
+         * Method that executes all migration from migration resources that are higher version than db version. If migration fails, method
+         * will exit.
```

Comment:
Doesn't it throw and error if the response doesn't satisfy required consistency?

Reply:
If required consistency level isn't satisfied exception will be thrown which will cause migration execution to fail. This comment was there before, I haven't changed anything except formatting.